src/lib/public/links/links.json

```diff
@@ -58,5 +58,6 @@
   "hot": "/blog/871",
   "denokv": "https://docs.google.com/presentation/d/1G0xpa3naCXybyLy11cajmWafZ3PwMG-cwG_KgPYLY4g/edit?usp=sharing",
   "dd": "https://docs.google.com/presentation/d/19pFLUTXilS61O2RWzCaSsGlAQVbr7Mguff-mVIERyWc/edit?usp=sharing",
-  "api-workshop": "https://docs.google.com/presentation/d/1Ruega88qAsajW6DlzG3SRwsDAx9mTaUZxxTXgA3-xlU/edit?usp=sharing"
+  "api-workshop": "https://docs.google.com/presentation/d/1Ruega88qAsajW6DlzG3SRwsDAx9mTaUZxxTXgA3-xlU/edit?usp=sharing",
+  "5.1-rampup": "https://docs.google.com/presentation/d/1mU9QCgO8BtjpZ4jmw-hzDVp-_YeaJAb8b5xbD3YIPsc/edit?usp=sharing"
```

Comment:
> No trailing comma which is common for this file as per [@ec7fa3526bdec3f2e374786451ea9832b1f2b4e0](https://github.com/EthanThatOneKid/acmcsuf.com/commit/ec7fa3526bdec3f2e374786451ea9832b1f2b4e0)

Thanks for the suggestion! For the referenced commit, the automated format check failed because JSON files do not expect a trailing comma.

[![image](https://github.com/EthanThatOneKid/acmcsuf.com/assets/31261035/567ed94b-2b24-4fc7-9930-481de93fbeca)](https://github.com/EthanThatOneKid/acmcsuf.com/actions/runs/5270498511/jobs/9530084905#step:8:230)